src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -1,28 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import styles from './NewFurniture.module.scss';
 import ProductBox from '../../common/ProductBox/ProductBoxContainer';
 import ProductsCompare from '../ProductsCompare/ProductsCompareContainer';
+import SwipeComponent from '../../common/SwipeComponent/SwipeComponent';
 
 class NewFurniture extends React.Component {
   state = {
     activePage: 0,
     activeCategory: 'bed',
+    manualPageChange: false,
   };
 
   handlePageChange(newPage) {
-    this.setState({ activePage: newPage });
+    this.setState({ fade: true });
+    setTimeout(
+      () => this.setState({ activePage: newPage, fade: false, manualPageChange: true }),
+      100
+    );
   }
 
   handleCategoryChange(newCategory) {
-    this.setState({ activeCategory: newCategory });
+    this.setState({ fade: true });
+    setTimeout(
```

Comment:
Tak sie na to patrzę i wsumie to nie wiem, dlaczego umieściłeś tutaj `setTimeout` ?

Reply:
Łukasz, kombinowałem z przejściami pomiędzy stronami, ale w końcu dałem sobie spokój, niestety nie usunąłem tego nadmiarowego kodu. Własnie przesłałem poprawiony plik.